lego/apps/events/fixtures/development_events.yaml

```diff
@@ -4,39 +4,43 @@
     event_type: company_presentation
     location: TBA
     start_time: '2016-10-27T17:15:00+00:00'
-    text: "<p>TBA</p>"
+    text: '<p>TBA</p>'
     title: EINR
     is_priced: True
     price_member: 50
     price_guest: 100
     created_by: 3
     require_auth: False
     cover: test_event_cover.png
+    youtube_url: https://www.youtube.com/watch?v=bLHL75H_VEM
```

Comment:
Same as for dev articles. You're changing the data people are using for developing other things as well, so it might be better to add a separate event for youtube-stuff. However, there are like 50 events to use , compared to only 2 articles, so it's not really a big deal here.